messages/plugin-invalid.txt

```diff
@@ -0,0 +1,8 @@
+"<%= configName %>" is invalid syntax for a config specifier.
+
+* If your intention is to extend from a configuration exported from the plugin, add the configuration name after a slash: e.g. "<%= configName %>/myConfig".
+* If this is the name of a shareable config instead of a plugin, remove the "plugin:" prefix: i.e. "<%= configName.slice("plugin:".length) %>".
```

Comment:
TIL this is allowed in [`_.template`](https://lodash.com/docs/4.17.15#template). Confirmed locally this does what we want it to:

```

Oops! Something went wrong! :(

ESLint: 7.7.0

"plugin:wat" is invalid syntax for a config specifier.

* If your intention is to extend from a configuration exported from the plugin, add the configuration name after a slash: e.g. "plugin:wat/myConfig".
* If this is the name of a shareable config instead of a plugin, remove the "plugin:" prefix: i.e. "wat".

'"plugin:wat"' was referenced from the config file in "C:\Code\eslinttemp\.eslintrc.json".

If you still can't figure out the problem, please stop by https://eslint.org/chat/help to chat with the team.
```

Reply:
This output looks awesome!

I can also confirm locally that this works. It seems that `<-` and `<=` can evaluate code, although it doesn't look like that in the lodash docs.